src/device-api/v1.ts

```diff
@@ -46,84 +49,50 @@ router.post('/v1/restart', (req: AuthorizedRequest, res, next) => {
 		.catch(next);
 });
 
-const v1StopOrStart = (
-	req: AuthorizedRequest,
-	res: express.Response,
-	next: express.NextFunction,
-	action: 'start' | 'stop',
-) => {
-	const appId = checkInt(req.params.appId);
-	const force = checkTruthy(req.body.force);
-	if (appId == null) {
-		return res.status(400).send('Missing app id');
-	}
-
-	return Promise.all([applicationManager.getCurrentApps(), getApp(appId)])
-		.then(([apps, targetApp]) => {
-			if (apps[appId] == null) {
-				return res.status(400).send('App not found');
-			}
-			const app = apps[appId];
-			let service = app.services[0];
-			if (service == null) {
-				return res.status(400).send('No services on app');
```

Comment:
Same as above, this also became `404 Not Found` instead of `400 Bad Request` which is an interface change, but I believe is more correct.

Reply:
Reverted to original interface in https://github.com/balena-os/balena-supervisor/commit/98f4ded24f1c61816f8f01ffce318c1fb0aa8c8a. This means that for endpoints that stop/start a service, v1 endpoints will throw BadRequestError (400) for the same code paths that v2+ endpoints throw NotFoundError (404). This maintains the legacy interface.